sections/semantics-document-metadata.include

```diff
@@ -832,12 +832,11 @@
   <div class="example">
   The following examples illustrate code that should be avoided:
   <pre highlight="html">
+    &lt;!-- DO NOT DO THIS -->
```

Comment:
3 stray &lt;s - lines 835, 837, 839

Reply:
No, these are in a `pre` element, with live `<mark>` elements so specific bits get highlighted. So the code for display needs to be escaped as it is.

Reply:
Chaals is right. They need to stay escaped.